pkg/users/authorized-keys.js

```diff
@@ -79,12 +77,12 @@ function AuthorizedKeys (user_name, home_dir) {
                                         .join(" ") || null;
                         }
                     }
+                    return keys;
                 })
-                .always(function() {
-                    df.resolve(keys);
+                .catch(ex => {
+                    cockpit.warn("Failed to list public keys:", ex.toString());
+                    return [];
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18488-20230310-180230-a6731aad-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)